.github/workflows/newman.yml

```
@@ -41,8 +54,219 @@ jobs:
           echo "GITHUB_OUTPUT contents:"
           cat $GITHUB_OUTPUT
 
+  associate-test-cases:
+      runs-on: ubuntu-22.04
+      outputs:
+        test_exec: ${{ steps.xray-newman.outputs.test_exec }}
+        test_plan_id: ${{ steps.get-testPlan-id.outputs.testPlan_id }}

```

update variables for more consistency
```suggestion
        test_plan_id: ${{ steps.get-test-plan-id.outputs.test_plan_id }}
```